internal/usage/usage.go

```diff
@@ -71,6 +71,8 @@ const (
 	NotificationToken               = "Slack API token or Bot token or Flowdock personal API token." //nolint:gosec // This is just a message not a password
 	NotificationsChannelName        = "Slack channel name. Required for the SLACK notifications type."
 	APIKey                          = "Datadog API Key, Opsgenie API Key, VictorOps API key."
+	APIKeyDescription               = "Description of the API key."
+	APIKeyRoles                     = "List of roles for the API key."
```

Comment:
We can reuse `usage.Roles` here

Reply:
I initially thought this too, but the `usage.Roles` refers to databases/collections

Reply:
we are using `usage.Roles` for creating and updating organization api keys. I think that we should update them as well if we are going to use `APIKeyRoles` here

Reply:
Ok i'll merge master and update those now